manticore/core/smtlib/solver.py

```diff
@@ -583,6 +603,12 @@ def _optimize_fancy(self, constraints: ConstraintSet, x: BitVec, goal: str, max_
             self._smtlib.send("(%s %s)" % (goal, aux.name))
             self._smtlib.send("(check-sat)")
             _status = self._smtlib.recv()
+
+            assert self.sname is not None
+            if self.sname not in SOLVER_STATS:
+                SOLVER_STATS[self.sname] = 0
```

Comment:
For future reference, Python dicts include a builtin method for doing membership checking and initialization in one line. So, this line and the previous would become `SOLVER_STATS.setdefault(self.sname, 0)`